v2/src/core/components/TheAppFooter.vue

```diff
@@ -75,6 +75,14 @@
                             <v-icon class="ml-2">settings</v-icon>
                         </a>
                         <p class="mb-3">Pricing from Coingecko</p>
+                        <v-switch
```

Comment:
There is something weird that is happening on the background. Turn on dark mode. Then **reload** the page, switch back to light mode and you will see that toggle button is disappearing